src/lib/extractMultiplayerMetadata.ts

```diff
@@ -0,0 +1,25 @@
+const regexes = {
+  minPlayers: /minPlayers\s*:\s*?([0-9])/,
+  maxPlayers: /maxPlayers\s*:\s*?([0-9])/,
+  playerJoined: /playerJoined\s*[:(]/,
+  playerLeft: /playerLeft\s*[:(]/,
+}
+
+export function extractMultiplayerMetadata(logicJsContent: string) {
+  const minPlayersString = logicJsContent.match(regexes.minPlayers)?.at(1)
+  const minPlayers = minPlayersString ? parseInt(minPlayersString) : undefined
+
+  const maxPlayersString = logicJsContent.match(regexes.maxPlayers)?.at(1)
+  const maxPlayers = maxPlayersString ? parseInt(maxPlayersString) : undefined
+
+  const handlesPlayerJoined = regexes.playerJoined.test(logicJsContent)
```

Comment:
These checks feels not strict enough. What if there is a minPlayers/maxPlayers/playerJoined variable somewhere inside the actions?

Reply:
As we discussed, hopefully this is good enough in combination with runtime checks/review for now and it might be unnecessary in the future if we switch to some kind of game manifest file approach